<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParaDialogCombine</class>
 <widget class="QDialog" name="ParaDialogCombine">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atlas Guided C.elegans Cell Annotation</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>560</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_imgv3d">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Image and Marker (anchor) from V3D:</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_imgv3d">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>431</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioButton_atlasfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>175</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Atlas from File:</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_atlasfile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atlasfile">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>471</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Image, Atlas and Marker Import</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>221</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Initial Align paras:</string>
   </property>
   <widget class="QCheckBox" name="checkBox_wormupsidedown">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Worm upside down?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_refchannel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_refchannel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>reference channel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_downsampleratio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>downsample ratio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temperature_ini">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>initial temperature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temperature_min">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>minimal temperature:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_annealingrate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>annealing rate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_iter_pertemp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>iter per temperature:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_downsampleratio">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_temp_ini">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_temp_min">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_annealingrate">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_niter_pertemp">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(r:0, g:1, b:2)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fgthresh">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_annealingrate_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FG thresh factor:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>241</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Refine Align paras:</string>
   </property>
   <widget class="QCheckBox" name="checkBox_refinealign_simple">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Simple refined align</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>stop iteration when:</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>iter num &gt; </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ref_maxiter">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ref_minposchange">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>total pos channge &lt; </string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ref_cellradius">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>fixed temperature:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ref_temperature">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>cell radius:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>(fast but less accurate)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output deformed Atlas to:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_atlas_output">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atlas_output">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Visulization (realtime)</string>
   </property>
   <widget class="QCheckBox" name="checkBox_showV3D_pts">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>show deforming atlas in V3D</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>buttonBox</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_5</zorder>
  <zorder>groupBox_8</zorder>
  <zorder>groupBox_9</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ParaDialogCombine</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ParaDialogCombine</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_imgv3d</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_imgv3d</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
